app/controllers/scenarios_controller.rb

```diff
@@ -69,6 +70,7 @@ def edit
 
   def create
     @scenario = current_user.scenarios.build(params[:scenario])
```

Comment:
I just did. I set the `type` of the `icon` to string because I am storing only string names of the glyphs only. Should we do something for the existing instances out there or is this sufficient?